lightning/src/chain/keysinterface.rs

```diff
@@ -177,15 +177,29 @@ pub struct InMemoryChannelKeys {
 }
 
 impl ChannelKeys for InMemoryChannelKeys {
-	fn funding_key(&self) -> &SecretKey { &self.funding_key }
-	fn revocation_base_key(&self) -> &SecretKey { &self.revocation_base_key }
-	fn payment_base_key(&self) -> &SecretKey { &self.payment_base_key }
-	fn delayed_payment_base_key(&self) -> &SecretKey { &self.delayed_payment_base_key }
-	fn htlc_base_key(&self) -> &SecretKey { &self.htlc_base_key }
-	fn commitment_seed(&self) -> &[u8; 32] { &self.commitment_seed }
+	fn funding_key(&self) -> &SecretKey {
+		&self.funding_key
+	}
+	fn revocation_base_key(&self) -> &SecretKey {
+		&self.revocation_base_key
+	}
+	fn payment_base_key(&self) -> &SecretKey {
+		&self.payment_base_key
+	}
+	fn delayed_payment_base_key(&self) -> &SecretKey {
+		&self.delayed_payment_base_key
+	}
+	fn htlc_base_key(&self) -> &SecretKey {
+		&self.htlc_base_key
+	}
+	fn commitment_seed(&self) -> &[u8; 32] {
+		&self.commitment_seed
+	}
```

Comment:
Could you move `fn_single_line` to a separate commit for now? Turns out it doesn't just allow single-line functions but rather it forces them for all functions of single expressions since there is a large `max_width`.  The churn is making it difficult to review the other changes.

Reply:
oops, fixed!